packages/@sanity/types/src/schema/definition/type/string.ts

```diff
@@ -0,0 +1,25 @@
+import {RuleDef, ValidationBuilder} from '../../ruleBuilder'
+import {InitialValueProperty} from '../../types'
+import {BaseDefinitionOptions, EnumListProps} from './common'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface StringOptions extends EnumListProps<string> {}
```

Comment:
`StringOptions` might be a tad too generic. Could it be `StringSchemaOptions`?

Reply:
If we go that route, then everything should have SchemaOptions suffix.

Reply:
Tried toing the refactor, but this will cascade into a lot of code in a lot of packages. It will also be a painful search and replace for studios that use these types.

I guess we could let any options that already had a type remain as is, but that seems very inconsistent. 
I would argue that in the scope of `sanity` StringOptions is not THAT generic; of course we should not use those name anywhere else.

Reply:
I retract my feedback. Keeping `StringOptions` is fine 👍